server/spec/multi_org_spec.rb

```diff
@@ -679,4 +679,74 @@
     expect(@cp.get_pool(pool['id']).quantity).to eq(1)
     expect(@user_client.list_pool_entitlements(pool['id']).size).to eq(1)
   end
+
+  it 'uses a host-restricted pool from the sharing org' do
```

Comment:
werent we discussing that we would add a spec test where the host in another org is not healed so the heal of the guest fails? is that already covered somehow?